### DISCUSSION TITLE
question about tensorrt - batch > 1.

### DISCUSSION THREAD

**Discussion by YoungjaeDev** [obss/sahi#808] [Q&A]
1. Is trt-engine foward currently available regardless of batch size? 
2. Can you tell me the exact reason why only single batch inference is possible in the current state?
I think it will be helpful in implementing c++, so please reply.

**Comment by fcakyon** [comment]
CUrrent implementation has a limitation of 1 batch size. `sahi==0.12.0` will be fixing this limitation.

**Reply by luan1412167** [reply]
Hi @fcakyon, when sahi==0.12.0 with batch inference release?
